### ISSUE TITLE
Expose API for direct access to the context

### ISSUE THREAD

**Issue by puzpuzpuz** [fastify/fastify-request-context#13]
## 🚀 Feature Proposal

Expose request context (at least the CLS part of it) as a global object instead of (or as an alternative to) asking users to access it through `req`/`app`.

## Motivation

The whole purpose of any CLS is to avoid having to pass a context object through application modules. Exposing request context through the `req` object kinda kills the whole idea. Having it available on `app` makes things a bit better as it could provide isolation guarantees for multiple Fastify instances running in a single node process, but it looks like the underlying `asynchronous-local-storage` library has a single global `AsyncLocalStorage` instance, which means no isolation as such.

## Example

```js
const { ctx } = require('fastify-request-context')

// Later in route handler
const foo = ctx.get('bar')
```

**Comment by kibertoad** [comment]
Great idea! I'll add this later today.

**Comment by kibertoad** [comment]
@puzpuzpuz Do you think that isolation across multiple fastify instances is an important feature and access to context should be key-based, or just one instance is enough?

**Comment by puzpuzpuz** [comment]
In my opinion, majority of users are using a single Fastify instance per app, so a global request context should be fine for them. There might be cases when the app runs multiple Fastify instances (and thus listens on multiple ports), but those are quite rare.

**Comment by kibertoad** [comment]
Thanks, that makes sense!

**Comment by kibertoad** [comment]
Released in 2.1.0
